src/components/ProjectName/ProjectName.js

```diff
@@ -46,7 +46,7 @@ const ProjectName = () => {
       databaseSwitchboardInstance.getProject(projectId).then((projectResponse) => {
         if (isMounted.current) {
           setProjectName(projectResponse?.name)
-          setIsTestProject(projectResponse.status < 90)
+          setIsTestProject(projectResponse?.status < 90)
         }
```

Comment:
This fixes a bug where a user pastes a project url into the address bar when they dont have read access to that project